library/psa_crypto.c

```diff
@@ -6718,90 +6732,103 @@ static psa_status_t psa_jpake_output_prologue(
     psa_pake_operation_t *operation,
     psa_pake_step_t step)
 {
-    psa_jpake_computation_stage_t *computation_stage =
-        &operation->computation_stage.jpake;
-
-    if (computation_stage->state == PSA_PAKE_STATE_INVALID) {
-        return PSA_ERROR_BAD_STATE;
-    }
-
     if (step != PSA_PAKE_STEP_KEY_SHARE &&
         step != PSA_PAKE_STEP_ZK_PUBLIC &&
         step != PSA_PAKE_STEP_ZK_PROOF) {
         return PSA_ERROR_INVALID_ARGUMENT;
     }
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_JPAKE)
```

Comment:
This function is JPAKE specific thus what about to rather guard the whole functions with #if defined(MBEDTLS_PSA_BUILTIN_ALG_JPAKE) ? Ditto for `psa_jpake_output_epilogue()`. It is fine by me if you force push the last commit.

Reply:
Force-pushed fix.